planner/core/point_get_plan.go

```diff
@@ -281,23 +281,29 @@ func checkFastPlanPrivilege(ctx sessionctx.Context, fastPlan *PointGetPlan, chec
 	return nil
 }
 
-func buildSchemaFromFields(ctx sessionctx.Context, dbName model.CIStr, tbl *model.TableInfo, fields []*ast.SelectField) *expression.Schema {
+func buildSchemaFromFields(ctx sessionctx.Context, dbName model.CIStr, tbl *model.TableInfo, tblAlias model.CIStr, fields []*ast.SelectField) *expression.Schema {
```

Comment:
It better to `s/tableAlias/tblName/` to keep the same naming style with function `getNameValuePairs()`

Reply:
done